agent/config/config.go

```diff
@@ -118,13 +118,13 @@ const (
 	minimumNumImagesToDeletePerCycle = 1
 
 	// defaultCNIPluginsPath is the default path where cni binaries are located
-	defaultCNIPluginsPath = "/amazon-ecs-cni-plugins"
+	defaultCNIPluginsPath = "/usr/libexec"
 
 	// DefaultMinSupportedCNIVersion denotes the minimum version of cni spec required
 	DefaultMinSupportedCNIVersion = "0.3.0"
 
 	// pauseContainerTarball is the path to the pause container tarball
-	pauseContainerTarballPath = "/images/amazon-ecs-pause.tar"
+	pauseContainerTarballPath = "/var/lib/amazon-ecs-pause.tar"
```

Comment:
feels like `/var/lib/ecs/` is the right path for this

Reply:
I thought so at first as well... but the .tar file ended up in `var/lib`. This directory is the same when bottlerocket agent was moved out of the container.